### PR TITLE
First version

### PR DESCRIPTION
**Pull request by lulco**

### CODE REVIEW

**Comment by lulco** [comment]
add github actions

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "keywords": ["worker", "driver", "redis-proxy"],
+    "license": "MIT",
+    "require": {
+        "php": "^7.4 | ^8.0 < 8.2",
+        "tomaj/hermes": "^4.0",
+        "lulco/redis-proxy": "dev-master as 1.0.0",
+        "symfony/console": "^5.4 | 6.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.1"
```
Comment (lulco) [inline composer.json:13]: if no unit tests will be written, remove this

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "efabrica/hermes-extension",
+    "description": "Extension for tomaj/hermes",
+    "keywords": ["worker", "driver", "redis-proxy"],
+    "license": "MIT",
+    "require": {
+        "php": "^7.4 | ^8.0 < 8.2",
```
Comment (ricco24) [inline composer.json:7]: \>=7.4 <8.2

File: src/Command/HermesWorker.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    protected function configure(): void
+    {
+        $this->setDescription('Handle hermes messages');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->dispatcher->handle();
+        $output->writeln('Hermes worker end.');
+        return 0;
```
Comment (ricco24) [inline src/Command/HermesWorker.php:31]: return self::SUCCESS;

File: src/Driver/RedisProxySetDriver.php [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        return (bool)$this->redis->sadd($key, $this->serializer->serialize($message));
+    }
+
+    public function setupPriorityQueue(string $name, int $priority): void
+    {
+        $this->queues[$priority] = $name;
+    }
+
+    /**
+     * @throws RedisProxyException
```
Comment (ricco24) [inline src/Driver/RedisProxySetDriver.php:60]: Maybe wrap all driver specific exceptions to under one driver exception that can be handled in dispatcher?

File: composer.json (hunk 2 above)
Comment (lulco) [reply composer.json:7]: 👍

File: src/Driver/RedisProxySetDriver.php (hunk 4 above)
Comment (lulco) [reply src/Driver/RedisProxySetDriver.php:60]: there is nothing like catching Hermes exception in Dispatcher

```
} catch (Exception $exception) {
    if (Debugger::isEnabled()) {
        Debugger::log($exception, Debugger::EXCEPTION);
    }
}
```

File: src/Command/HermesWorker.php (hunk 3 above)
Comment (lulco) [reply src/Command/HermesWorker.php:31]: I thought it is only in symfony/console 6, added

**Pull request merged**